src/shared/modals/custom-filter/FilterTabs.js

```diff
@@ -27,6 +27,34 @@ const FilterTabs = (props) => {
     setDisplayedRows
   } = props
 
+  const renderFilterErr = () => {
+    return (
+      <div className="filter__search-error">
+        <div>No Results</div>
+        <p>Sorry we didn’t find any row titles that match your search.</p>
+      </div>
+    )
+  }
+
+  const renderDisabledSelect = () => {
+    return (
+      <div className="checkbox__container checkbox__container--select-all">
+        <input
+          readOnly
```

Comment:
Why is the readOnly attribute used here? I would think that disabled would make more sense.